### PR TITLE
Adds stub for ServiceEntityRepository

### PR DESCRIPTION
**Pull request by gianiaz**
This stub was missing, hope it helps :-)

### CODE REVIEW

File: extension.neon [hunk 1]
```diff
@@ -22,6 +22,8 @@ parameters:
 		- stubs/ObjectManager.stub
 		- stubs/ObjectManagerDecorator.stub
 		- stubs/ObjectRepository.stub
+		- stubs/ObjectRepository.stub
```
Comment (ondrejmirtes) [inline extension.neon:25]: This stub no longer exists, probably wrong rebase.

Comment (gianiaz) [reply extension.neon]: Sorry, I think I've fixed

**Comment by ondrejmirtes** [comment]
Thank you!

**Comment by Tobion** [comment]
I assume this change makes phpstan complain

> Class Acme\MyRepository extends generic class Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository but does not specify its types: TEntityClass

@ondrejmirtes @gianiaz what annotation do I need to put on MyRepository to make phpstan know the "TEntityClass"?
I tried `@implements ServiceEntityRepository<MyEntity>` but that wasn't it.

**Comment by Tobion** [comment]
Somewhat related to #67

**Comment by ondrejmirtes** [comment]
You’re extending a class so you need `@extends`. Here’s the docs: https://phpstan.org/blog/generics-in-php-using-phpdocs

**Comment by Tobion** [comment]
Thanks @ondrejmirtes

**Comment by Kocal** [comment]
Following this PR and #67, is there a way to automatically define `TEntityClass` with the 2nd argument of `ServiceEntityRepository::__construct()` through a PHPStan extension?

This way users won't have to `@extends ServiceEntityRepository<MyClass>` manually.

Thanks!

**Comment by ondrejmirtes** [comment]
@Kocal It's not feasible to do something like that in an extension, having `@extends` annotation is required. Already answered here: https://github.com/phpstan/phpstan-doctrine/issues/178

**Comment by Kocal** [comment]
Alright, sorry for not seeing the response before.
Thanks!

**Pull request merged**